genienlp/metrics.py

```diff
@@ -179,25 +179,31 @@ def computeNMTBLEU(outputs, targets):
     return bleu_metric.compute(predictions=outputs, references=targets)['bleu'] * 100
 
 
-def compute_e2e_dialogue_score(greedy, answer, tgt_lang, args, example_ids):
+def compute_e2e_dialogue_score(greedy, answer, tgt_lang, args, example_ids, contexts):
```

Comment:
I think we should use `predictions` and `answers` (plural) instead of `greedy` and `answer` (like in `compute_ner_f1`). Makes it less confusing if someone is not using greedy decoding.

Reply:
Yeah, we need to update all function signatures in metrics to be consistent. Right now we have `(outputs, targets)`, `(greedy, answer)`, or `(predictions, answers)`. I added it to the cleanup TODOs: